UnityProject/Assets/Scripts/Managers/PlayerInfo.cs

```diff
@@ -162,7 +167,18 @@ private void TryChangeName(string playerName)
 
 	public void SetMind(Mind InMind)
 	{
+		if (Mind != null)
```

Comment:
I don't get what's going on here, can you explain why are you doing this?

Reply:
it's removing itself from the previous mind it was on

Reply:
righto then, carry on.